src/main/default/lwc/lookup/lookup.js

```diff
@@ -374,4 +391,8 @@ export default class Lookup extends LightningElement {
     get isExpanded() {
         return this.hasResults();
     }
+
+    get hasMoreResultsToDisplay() {
+        return this._hasFocus && this._searchResultsCount > this.resultsToDisplay;
```

Comment:
can you explain how search result counts are related to hasFocus? maybe add a note if this is really needed

Reply:
we only want to display the helper text when the user is searching for something, as soon as he moves away from the input we don't want to display the text